### DISCUSSION TITLE
Single view of items

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#10487] [Feature Requests]
One thing is to navigate and see items per collection, another is to have one single view with common values, for example, ID as every item should have at least one ID.

Note: I checked and I could not find a duplicate idea.

**Comment by rijkvanzanten** [comment]
I'm not sure I understand what you mean.. Do you have a practical example to share?

**Reply by juancarlosjr97** [reply]
Sure, thinking about how Drupal can show records of a different type in a single view. Imaging update multiple records, different types, unrelated or viewing last updated regardless their type.

If I have a system with 10 collections, each unique and unrelated but I want to see the last updated, I will have to check each one of them.

**Reply by rijkvanzanten** [reply]
Ahh gotcha. You could use an insights dashboard with the list panel for that!

**Reply by juancarlosjr97** [reply]
Yes, I know that but requires manual work every time after adding a new collection. I still consider a single view by default would be something good as one view with easy scrolls, instead of dashboards.

**Reply by benhaynes** [reply]
Maybe you could pull insights from activity or revisions, since those automatically receive item updates from all collections?

**Reply by juancarlosjr97** [reply]
I will have to play with it more, but as I could see a list as a collection view is not possible, I can be mistaken though

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
